sqlglot/parser.py

```diff
@@ -4315,7 +4315,9 @@ def _parse_primary(self) -> t.Optional[exp.Expression]:
 
             this = self._parse_query_modifiers(seq_get(expressions, 0))
 
-            if isinstance(this, exp.UNWRAPPED_QUERIES):
+            if not this and self._match(TokenType.R_PAREN, advance=False):
```

Comment:
why was this added?

Reply:
It was indeed a hacky solution in order to not leak even more complexity into the parser. 

Although the Snowflake docs show that `NULL_IF` should have at least one string value i.e `NULL_IF = ('<string'> [, '<string>' ...])`, I came across [threads such as this one](https://community.snowflake.com/s/article/how-to-update-file-format-with-nullif-to-remove-n-null-values-when-using-copy-into-location) which showcase that it can be used as is. So, the ideas I had at hand were to:

- Parse it as empty tuple
- Make `exp.Paren` args optional, so that `exp.Paren(this=None)` can generate empty parentheses
- Parse `NULL_IF` manually by looking ahead tokens, stitching together strings etc

I think the least troublesome (which is not totally wrong semantically wise ?) was the first option